hdr_html_canvas_element.md

```diff
@@ -0,0 +1,744 @@
+
+#### Scene light
+
+Scene light is light that is captured by a capture device (e.g, a camera) at each pixel.
+Scene light also has a maximum value, either imposed by the limitations of the capture device, or the known maximum value of the particular scene being captured.
+
+The precise units of scene light are vague.
+Under some circumstances, the units are precise number of nits of each input pixel.
+Under some circumstances, the units are normalized to the interval [0, 1].
+We will use the latter convention, with scene light being normalized such that the value of 1 represents diffuse white, and we will draw attention to situations where we do not follow this rule.
+
+Similarly to display light, scene light is sometimes referred to as being "linear".
+
+#### Signal
+
+A signal is a digial encoding of scene light.
+
+An example of a signal is the pixel values in an sRGB image.
+
+Most signals take values in the [0, 1] interval and are represented using fixed-point (e.g, the usual 8-bit encoding of an sRGB image).
+In this document, we will be allowing for signals that take any real value, and are represented using floating-point.
+
+### Transfer functions
+
+#### Opto-electronic transfer function (OETF)
+
+An opto-electronic transfer function (OETF) is a function that transforms scene light to a signal.
+
+An example of an OETF is the [sRGB function](https://www.w3.org/Graphics/Color/srgb) ``f(x)=(x<0.0031308) ? 12.92*x : 1.055*pow(x,1/2.4)-0.055``.
+The domain of this OETF (scene light) is the interval [0, 1], with diffuse white at 1 (which is what we adopted as convention).
+The range (signal) is standard sRGB pixel values in the interval [0, 1].
+
+Another example of an OETF is the HLG OETF, specified in Table 5 of the [BT.2100 specification](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-2-201807-I!!PDF-E.pdf).
+For this OETF, the domain (scene light) is normalized to the interval [0, 1], with maximum brightness at 1 (which we will treat as matching our convention).
+
+Another example of an OETF is the PQ OETF, specified in Table 4 of the [BT.2100 specification](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-2-201807-I!!PDF-E.pdf).
+For this OETF, the domain (scene light) is luminance values in nits (which does not not match our convention, and we will discuss how to match this in detail).
+
+#### Opto-optical transfer function (OOTF)
+
+An opto-optical transfer function (OOTF) is a function that transforms scene light to display light.
+
+It is the responsibility of the OOTF to reproduce the appearance of the scene, within the capabilities of the display device, and subject to viewer's environment.
+
+The HLG OOTF is parameterized by the maximum luminance of the display device. It maps a fixed range of scene light to cover the full range (up to maximum luminance) of the display device.
+
+The PQ OOTF can be thought of as the identity function, with scene light equaling display light.
```

Comment:
Is this correct? BT.2100 defines the OOTF as approx. a gamma 1.2 applied on RGB.